app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -379,7 +388,14 @@ String licenseLink(Media media) {
 
     private void openWebBrowser(String url) {
         Intent browser = new Intent(Intent.ACTION_VIEW, Uri.parse(url));
-        startActivity(browser);
+        //check if web browser available
+        if(browser.resolveActivity(getActivity().getPackageManager()) != null){
```

Comment:
Please use consistent whitespaces. :) We usually use a whitespace between `if` and `(`, similar to what you did in the other classes. Same with `(` and `{`.